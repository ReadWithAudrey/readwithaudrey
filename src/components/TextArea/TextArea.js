import React from 'react'
import { StyledTextArea } from './TextArea.style'
import PropTypes from 'prop-types'

const TextArea = props => (
  <StyledTextArea
    placeholder={props.placeholder}
    onChange={props.onChange}
    name={props.name}
    rows="5"
  />
)

TextArea.propTypes = {
  placeholder: PropTypes.string,
  onChange: PropTypes.func,
  name: PropTypes.string,
}

export default TextArea
