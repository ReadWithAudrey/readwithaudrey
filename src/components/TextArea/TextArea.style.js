import styled from 'styled-components'

export const StyledTextArea = styled.textarea.attrs({
  className: 'montserrat',
})`
  background: #fff;
  border: #d3d3d3 1px solid;
  padding: 7px;
  margin: 10px 0px;
  border-radius: 10px;
  width: 100%;
`
