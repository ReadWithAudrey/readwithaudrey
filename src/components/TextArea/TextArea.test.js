import React from 'react'
import renderer from 'react-test-renderer'
import TextArea from './TextArea'

describe('TextArea component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<TextArea />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
