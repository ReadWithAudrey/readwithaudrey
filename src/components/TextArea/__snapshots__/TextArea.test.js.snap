// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea component snapshot test renders correctly 1`] = `
<textarea
  className="sc-bdVaJa montserrat jqMHOR"
  rows="5"
/>
`;
